pkg/token/token.go

```diff
@@ -172,8 +172,11 @@ func (g generator) GetWithRole(clusterID string, roleARN string) (string, error)
 	// if a roleARN was specified, replace the STS client with one that uses
 	// temporary credentials from that role.
 	if roleARN != "" {
+		opt := func(p *stscreds.AssumeRoleProvider) {
+			p.Duration = time.Duration(duration) * time.Minute
```

Comment:
The usage message above says "Seconds" but this looks like the argument is interpreted as minutes? I think passing a `time.Duration` all the way through (rather than `int`) might be better to avoid this type of confusion.

Reply:
The usage is wrong. I intended "Minutes" 🙇